examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_view_offset_consistency.py

```diff
@@ -0,0 +1,188 @@
+        :return: numpy array of (x,y,z)
+        """
+        coord = line[:-2].split(" ")
+        return np.array([float(coord[0]), float(coord[1]), float(coord[2])])
+
+
+    def __extract_coords_from_lines(self, lines):
+        """
+        Assumes that lines comes in as a string with three numbers, each split by a space.
+        Iterates through each file to get coordinate data for each entry.
+        First line is original coordinate of the point source and
+        following lines are the closest voxel to the origin for each LOR in the root file.
+        :param lines: Input lines, loaded from the file
+        :return: A tuple of the original coordinate numpy array and a list of coordinates of closes voxels in the LOR
+        """
+        is_first = True
+        coord_list = []
+        for line in lines:
+            if is_first:
+                original_coord = self.__process_line(line)
+                is_first = False
+                continue
+            coord_list.append(self.__process_line(line))
+        return (original_coord, coord_list)
+
+    def compute_distance_to_original(self, axis=None):
+        if axis is None:
+            # Compute l2
+            return [np.linalg.norm(self.original_coord - coord) for coord in self.coord_list]
+        else:
+            # compute distance in 1 dimension, but extends negative
+            return [coord[axis] - self.original_coord[axis] for coord in self.coord_list]
+
+    def get_num_events(self):
+        return len(self.coord_list)
+
+    def get_num_failed_events(self, tolerance=None):
+        if tolerance is not None:
+            print(f"Overwriting tolerance value as {tolerance}")
+            self.tolerance = tolerance
+        return sum(errs > self.tolerance for errs in self.err_l2)
+
+
+def print_pass_and_fail(allowed_fraction_of_failed_events=0.5):
+    """
+    Prints the number of events, how many events failed and the percentage
+    :param allowed_fraction_of_failed_events: Fraction of events that could "fail" before the test failed
+    :return: None
+    """
+    print(f"\n\nInformation regarding pass fail rate of the view offset test\n"
+          f" ID || The Number of Events  ||  The Number of Failed Events  ||  Percentage Error\n"
+          f"-----------------------------------------------------------------------------------")
+    for key in point_sources_data.keys():
+        num_events = point_sources_data[key].get_num_events()
+        num_failed_events = point_sources_data[key].get_num_failed_events()
+        percentage = num_failed_events / num_events * 100
+        if percentage > allowed_fraction_of_failed_events * 100:
+            warning_msg = "HIGH VALUE WARNING!"
+        else:
+            warning_msg = ""
+        print(f"{key}  ||           {num_events}         "
+              f"||              {num_failed_events}                "
+              f"||      {round_sig(percentage, 3)}      {warning_msg}")
```

Comment:
if you happen to know how to put consistent width in printing in Python, would be good to add it here